frontend/src/i18n/locales/de.yaml

```diff
@@ -18,7 +18,14 @@ general:
   leave-page-confirmation: >
     Seite verlassen? Vorgenommene Änderungen sind möglicherweise noch nicht gespeichert!
   logo-alt: Das Logo von „{{title}}“
+  no-root-children: Noch keine Seiten...
 
+welcome:
+  title: Willkommen zu Tobira!
+  body: >
+    Die Hauptseite scheint noch leer zu sein! Um Inhalte oder weitere Seiten
+    hinzuzufügen, melden Sie sich zuerst mit einem Account (mit Moderator-Rechten)
```

Comment:
I would would remove the parentheses here, like in the English version. It's essential information, after all.

Reply:
```suggestion
    hinzuzufügen, melden Sie sich zuerst mit einem Account mit Moderator-Rechten
```